### DISCUSSION TITLE
Custom environment with Gazebo simulation

### DISCUSSION THREAD

**Discussion by curieuxjy** [sail-sg/envpool#160] [Q&A] [answered]
Hello,

Can I use it with the custom environment with Gazebo simulation?
I couldn't find the documents about custom environments at a glance.
Am I missing or isn't it compatible with custom Gazebo env?

**Comment by Benjamin-eecs** [comment, marked as answer]
Hi, you can follow instructions on section [Customized C++ environment integration](https://envpool.readthedocs.io/en/latest/content/new_env.html) in the docs.

**Reply by curieuxjy** [reply]
Thanks a lot! :)
